src/components/pageContainer/PageContainer.css

```diff
@@ -0,0 +1,86 @@
+.page-container {
```

Comment:
I based this off your first page, then modified it to provide generic preset styles for any page (like styling of padding, scrolling, margins, tables, <sup>, lists, and images).